File: Python/nameBlocks.py
name = "Philip Bell" #setes the verable name to my name
incrementor = 2 #sets how many letter to print on each line
for i in range(0,len(name),incrementor): #loops through the following 
        try: #tries to do the following
            print name[i] + name[i+1] #prints out the current value in the list and the next one
        except IndexError: #if it runs off the end of the array and causes a IndexError then it does the following
            print name[i] #print the current value in the array
            

File: C#/nameBlocks.cs
using System;

namespace nameBlock
{
    class Program
    {
        static void Main(string[] args)
        {
            string name = "Philip Bell";
            int nameLength = name.Length;
            for (int i = 0; i < nameLength; i += 2)
            {
                char temp1 = name[i];
                if (i < nameLength-1)
                {
                    char temp2 = name[i + 1];
                    Console.Write(temp1);
                    Console.WriteLine(temp2);
                }
                else
                {
                    Console.WriteLine(temp1);
                }

            }
        }
    }
}
